client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -146,9 +247,19 @@ class ListItem extends PureComponent {
 
   preventDefault = event => event.preventDefault();
 
-  handleNameFocus = () => this.setState({ isNameEdited: true });
+  handleNameFocus = () => this.setState({ isNameEdited: true, busy: true });
+
+  handleNameBlur = () => this.setState({ isNameEdited: false, busy: false });
 
-  handleNameBlur = () => this.setState({ isNameEdited: false });
+  handleDescriptionFocus = () => this.setState({ busy: true });
+
+  handleDescriptionBlur = () => this.setState({ busy: false });
+
+  handleCommentsBlur = () => this.setState({ busy: false });
```

Comment:
I'm not sure if comments should be blocked, they are save in separate collection so I think it won't cause errors. Writing comment can take a long time so for all this time item will be locked. It will be very uncomfortable and annoying. Voting also, but voting is a very quick action so it won't cause big discomfort. But if you think about the list with several dozen users and only one of them can add a comment...